modules/categories/categories.model.js

```diff
@@ -0,0 +1,18 @@
+const categoryType = ` 
+    _id: ID!
+    categoryCode: String!
+    name: [Language!]
```

Comment:
This means that the list itself can be null, but it can't have any null members - is this the intention?

Reply:
yes